### ISSUE TITLE
Testing similarity Model

### ISSUE THREAD

**Issue by neerajhbhat** [Darthholi/similarity-models#3]
Hi,

Is there a way by which I can test the model on my own custom invoices.?

**Comment by holimart** [comment]
Of course, just create your own database with a similar schema filled with data from your custom invoices.
